src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,43 @@
+import cn from 'classnames';
+import { useContext } from 'react';
+import { DispatchContext } from './TodosContext';
+import { Status } from '../types/Status';
+
+export const TodoFilter = () => {
+  const dispatch = useContext(DispatchContext);
+
+  const filters = Object.values(Status);
+
+  const handleFilter = (filterBy: Status) => {
+    dispatch({
+      type: 'filter',
+      payload: filterBy,
+    });
+  };
+
+  return (
+    <ul className="filters" data-cy="todosFilter">
+      {
+        filters.map(filter => {
+          const filterName = filter[0].toUpperCase() + filter.slice(1);
+
+          return (
+            <li key={filter}>
+              <a
+                href={`#/${filter === 'all' ? '' : filter}`}
+                className={
+                  cn({
+                    selected: true,
```

Comment:
you should add this class only to the active filter
<img width="347" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/2206eae8-27b5-432a-bd9b-78a3bc394086">